app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,25 @@
+class FollowsController < ApplicationController
+
+    def create
+        @follow = current_user.relationships.new(follow_params)
+        if @follow.save
```

Comment:
![image](https://user-images.githubusercontent.com/94411241/182772648-48bea887-b9c9-4452-8c24-1a6c5c7199dd.png)
 Try this if it works, naduplicate na kasi yung redirection

Reply:
nagwork yung unless, already pushed na rin yung change sa code 😸